newsfragments/46613.feature.rst

```diff
@@ -0,0 +1 @@
+Operator Links interface changed to not run user code in Airflow Webserver The Operator Extra links, which can be defined either via plugins or custom operators now do not execute any user code in the Airflow Webserver, but instead push the "full" links to XCom backend and the value is again fetched from the XCom backend when viewing task details in grid view.
```

Comment:
Can you give some more info/short example code about what users who have a custom link class written have to do?

Reply:
Yeah can do that. Right now, there's a precommit limitation that the feature.rst can only be a single line long. Let me push a fix for that.

Reply:
Yeah can do that. Right now, there's a precommit limitation that the feature.rst can only be a single line long. Let me push a fix for that.

Reply:
Tracking via this: https://github.com/apache/airflow/pull/46681